src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Timestamp.cs

```diff
@@ -104,7 +104,11 @@ public Timestamp(SignedCms timestampCms)
             HashAlgorithmName fingerprintAlgorithm,
             List<SignatureLog> issues)
         {
-            settings = settings ?? SignedPackageVerifierSettings.GetDefault();
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
```

Comment:
style: You can use null coalescing here as well.

Reply:
null coalescing here will look something like `settings = settings ?? throw new ArgumentNullException` ... since I don't have to set settings, a null check seems like a better option.